device/mellanox/x86_64-mlnx_msn4800-r0/ACS-MSN4800/sai_4800.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
Did you get this file from SAI team? seems not like a regular one?

Reply:
I took the same file from 4700, I just removed all the ports on <ports-list> and changed the <number-of-physical-ports> to 0